src/data-table/column-numerical.js

```diff
@@ -338,6 +352,13 @@ function NumericalFilter(props) {
                 margin: '0 -7px',
               }),
             },
+            InnerTrack: {
+              style: ({$theme}) => ({
+                // For range selection we use the color as is, but when selecting the single value,
+                // we don't  want the track to standout so we mute its color
+                background: isRange ? undefined : theme.colors.mono400,
```

Comment:
Looks like this is backwards based on https://deploy-preview-4775--baseweb.netlify.app/?story=data-table--numerical-column

Reply:
ahh.. thanks for catching, glad I added the visual snapshot. Updating the visuals.